### PR TITLE
Render improvements for CraftingMonitorElements and CraftingPreviewElements

### PR DESCRIPTION
**Pull request by way2muchnoise**
Adds fluids to the Preview and gets rid of the weird generic to internal class structure of the rendering

### CODE REVIEW

File: src/main/java/refinedstorage/apiimpl/util/Comparer.java [hunk 1]
```diff
@@ -18,7 +18,9 @@ public boolean isEqual(ItemStack left, ItemStack right, int flags) {
         }
 
         if ((flags & COMPARE_OREDICT) == COMPARE_OREDICT) {
-            return isEqualOredict(left, right);
+            if (isEqualOredict(left, right)) {
```
Comment (InusualZ) [inline src/main/java/refinedstorage/apiimpl/util/Comparer.java]: why not `return isEqualOredict(left, right);` ?

Comment (InusualZ) [inline src/main/java/refinedstorage/apiimpl/util/Comparer.java]: nvm, I see why

Comment (way2muchnoise) [inline src/main/java/refinedstorage/apiimpl/util/Comparer.java]: If items aren't registered in the ore dict that will return false even if both stacks are the same Item, Damage and NBT

**Comment by acidjazz** [comment]
lets get a release going! :)

**Pull request merged**